ui-tests/package.json

```diff
@@ -20,7 +20,7 @@
   "author": "Project Jupyter",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "@playwright/test": "^1.31.0",
+    "@playwright/test": "^1.52.0",
```

Comment:
This seems to be fixing https://github.com/voila-dashboards/voici/issues/145

![image](https://github.com/user-attachments/assets/95def366-663a-477f-b07e-8704eb455bcd)